### PR TITLE
Make plot point size configurable

### PR DESCRIPTION
**Pull request by Oblarg**
my elderly aching eyes can't pick out the dead-pixel-like isolated points under the current settings

### CODE REVIEW

**Comment by Piphi5** [comment]
Can you send screenshots of the new plots?

**Comment by calcmogul** [comment]
Suggestion from the FRC Discord: since smaller point sizes hide less information and accurately present noise, we'd want to support both sizes. We could add a point size slider to the diagnostic plots window with a range of 1 to 2 (1 by default). It would go in the same block of code as the R^2 so it's hidden when the plots aren't in use.

**Comment by Oblarg** [comment]
image

**Comment by calcmogul** [comment]
You should rebase. The plot resizing only provides room for three text boxes, so there will be a scroll bar.

**Comment by Oblarg** [comment]
I just did rebase.

**Comment by calcmogul** [comment]
Seems like the "available area" automatically decreased by putting a slider before the plots. Neat.

File: sysid-application/src/main/native/include/sysid/view/AnalyzerPlot.h [hunk 1]
```diff
@@ -112,6 +112,8 @@ class AnalyzerPlot {
    */
   double* GetSimRSquared();
 
+  float m_pointSize = 1.25;
```
Comment (calcmogul) [inline sysid-application/src/main/native/include/sysid/view/AnalyzerPlot.h:115]: We generally put variables first, then functions in a given access specifier block.

**Review by calcmogul — CHANGES_REQUESTED** [review]

**Review by calcmogul — APPROVED** [review]

**Pull request merged**